sync_test.go

```diff
@@ -0,0 +1,237 @@
+package swarm
```

Comment:
This file is probably committed by mistake?

I wrote this test only to manually debug the issue. It is not a test that does any validation and contains a lot of print function calls. It surely is not in the mergeable condition.